geode-core/src/main/java/org/apache/geode/Delta.java

```diff
@@ -53,7 +51,13 @@
    * This method throws an {@link InvalidDeltaException} when the delta in the {@link DataInput}
    * cannot be applied to the object. GemFire automatically handles an {@link InvalidDeltaException}
    * by reattempting the update by sending the full application object.
-   *
    */
   void fromDelta(DataInput in) throws IOException, InvalidDeltaException;
+
+  /**
+   * Allows Delta implementations to ensure bucket sizes are recalculated after delta is applied
```

Comment:
Since this method is part of the external API improve these javadocs.
1. Add a @since 1.14
2. Explain what geode does for delta size by default (it is calculated once when the entry is first created and any changes in actual size will not be noticed by geode. This can cause the statistics that report memory use to not be accurate), why this default was chosen (to optimize the use case in which the actual size of the object implementing Delta does not change), how to override this default behavior (implement this method in your class that implements Delta and have it return true), and explain why you would want to do that (if your Delta causes the actual size of the instance implementing Delta to change). It would also be good to point out that changing this default behavior to recalculate may hurt performance.

Reply:
Missed the "@SInCE 1.14" the first time, adding now.